CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to `dash-ag-grid` will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 Links "DE#nnn" prior to version 2.0 point to the Dash Enterprise closed-source Dash AG Grid repo
 
+## UNRELEASED
+
+### Fixed 
+- [#262](https://github.com/plotly/dash-ag-grid/issues/262) Fixed all-but-one `cellValueChanged` events suppressed for multi-cell edits
```

Comment:
Rather than `### Fixed` this should be a `### Changed` with a very prominent note that it's a breaking change. So it's good that we're about to release a new major version for AG Grid v31 or this kind of change would not be allowed 😅 But it makes sense, I don't see any non-breaking way to support these events.

@BSd3v this makes me wonder, are there any other props that might have similar problems? If so now is the time to convert them into arrays too!

Reply:
If backwards compatibility is a main concern, we could

* Set the type of `cellValueChanged ` to union of single/list of events
* Add a new prop that represents `CELL_VALUE_CHANGED_DEBOUNCE_MS `, which defaults to -1, in which case we fallback to the old behavior, thus returning a single element. Setting the value it to 0 (or higher) will enable the new behavior, thus returning a list 

My initial implementation was actually along these lines, but @BSd3v suggested to target a simpler implementation (the one in this PR) in favor of backwards compatibility since a major release is in the making.

Reply:
Yep, I agree the simpler API is more important than backward compatibility, given the upcoming major release and the fact it would be annoying to use the prop if sometimes it’s a list and sometimes not. Just make sure the changelog makes the breaking change clear.

Reply:
I have updated the comment, indicating that this is a _change_. Should I note explicitly that the change is breaking (I am not sure how breaking changes are documented in this project - I didn't see any mentioned of breaking changes previously in the changelog)?

Reply:
This would be the first since switching to version 2 from 1.